code/game/machinery/camera/presets.dm

```diff
@@ -68,17 +68,11 @@
 //This camera type automatically sets it's name to whatever the area that it's in is called.
 /obj/machinery/camera/autoname/New()
```

Comment:
Change to initialize while you're here

Reply:
👍 , what is the difference?

Reply:
New happens outside our control at server start and switching to initialize lets us actually manage how and when it happens